.gitignore

```diff
@@ -7,3 +7,4 @@ npm-debug.log
 build
 website/core/metadata.js
 website/src/jest/docs
+.idea
```

Comment:
can you remove this and create a global gitignore on your computer that ignores `.idea` files? :)